app/queries/gobierto_people/people_with_activities_query.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
```

Comment:
I've been reviewing this class during at least 10 minutes and I don't fully follow the queries you are generating. I think you could include some comments with the queries the method `activities_sql` generates

Reply:
I've added some comments